instance/instance.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/cozy/cozy-stack/couchdb"
 	"github.com/cozy/cozy-stack/couchdb/mango"
 	"github.com/cozy/cozy-stack/vfs"
+	"github.com/dchest/uniuri"
+	scrypt "github.com/elithrar/simple-scrypt"
```

Comment:
I'm not really a fan of having dependencies like that to handle security in our code. Even if we copy/paste the library (or what's needed from it) it should *in-fine* be in our source tree and git repo.

We should be able to audit our code for security without having to go on github or any external source..